.travis.yml

```diff
@@ -21,4 +21,4 @@ before_script:
   - if [ "$deps" == "low" ]; then composer update --prefer-source --prefer-lowest --prefer-stable; fi
   - if [ "$deps" != "low" ]; then composer install --prefer-source; fi
 
-script: ./vendor/bin/phpunit
+script: composer test
```

Comment:
I think it would be more readable to have multiple commands here on Travis, so that they all run all the time, instead of running the next one only when the first one works